opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -362,10 +360,11 @@ impl Debug for UserEventsExporter {
 
 impl opentelemetry_sdk::logs::LogExporter for UserEventsExporter {
     async fn export(&self, batch: opentelemetry_sdk::logs::LogBatch<'_>) -> OTelSdkResult {
-        for (record, instrumentation) in batch.iter() {
-            let _ = self.export_log_data(record, instrumentation);
-        }
-        Ok(())
+        let (record, instrumentation) = batch
+            .iter()
+            .next()
+            .expect("batch is expected to have one and only one record");
```

Comment:
Should we use `expect()` here or be more defensive and use `if let Some(item)` construct? We know that we would never send an empty batch, but it might be sensible to still add this check in the exporter?

Reply:
I would be in favor of doing error handling here. It also aligns with our design principle:

https://github.com/open-telemetry/opentelemetry-rust/blob/main/docs/design/logs.md#key-design-principles

> Robust error handling, returning Result where possible instead of panicking.

Reply:
agree. made some change in latest commit to reflect this, PTAL.